tools/output-number-of-buffer-segments.rb

```diff
@@ -0,0 +1,49 @@
+#!/bin/ruby
+
+require "open3"
+require "json"
+require "date"
+
+if ARGV.size() == 0
+  puts "Usage: $0 DATABASE_PATH"
+  puts " e.g.: $0 ~/database/db"
+  exit
+end
+
+DATABASE_PATH = ARGV[0]
+ERROR_LOG_NAME = "output_n_buffer_segments.error"
+
+def output_error(status, stdout, stderr)
+  File.open(ERROR_LOG_NAME, 'a') do |file|
+    file.puts("#{Time.now}: #{status.inspect}, #{stdout.chomp}, #{stderr.chomp}")
+  end
+end
+
+stdout, stderr, status = Open3.capture3("groonga",
+                                        DATABASE_PATH,
+                                        "dump",
+                                        "--dump_plugins","no",
+                                        "--dump_schema", "no",
+                                        "--dump_records", "no",
+                                        "--dump_configs", "no")
+if (status.success?)
```

Comment:
```suggestion
if status.success?
```

We can reduce indent by:

```ruby
unless status.success?
  output_error(status, stdout, stderr)
  exit(false)
end

Reply:
Excuse me for cutting in.

How about exiting with `exit(status.exit status)` if we need return the exit status of this script?